cdci_osa_plugin/osa_common_pars.py

```diff
@@ -113,11 +120,16 @@ def __init__(self,
     def value(self):
         return self._value
 
+
     @value.setter
     def value(self, v):
         if v is not None:
             osa_version_base, osa_subversion, version_modifiers = split_osa_version(v)
 
+            if osa_version_base in self._obsolete_base_osa_version_values:
+                raise RequestNotUnderstood("Please note OSA11.0 is being phased out. "
+                                   "We consider that for all or almost all likely user requests OSA11.1 shoud be used instead of OSA11.0.")                                    
```

Comment:
Minor, I would write: "Please note `{{osa_version_base}}` is being..."

Reply:
Yes, it could be more generic.